okta-hosted-login/App.js

```diff
@@ -14,13 +14,13 @@ import React from 'react';
 
 import { Button, StyleSheet, Text, View } from 'react-native';
```

Comment:
Is this `okta-hosted-login/App.js` file really required for  RN sample app?

Reply:
`okta-hosted-login` will be deleted right after this get merged XD, since it contains old sample